crashreporter/src/main/java/com/tealium/crashreporter/CrashReporter.kt

```diff
@@ -0,0 +1,41 @@
+package com.tealium.crashreporter
+
+import com.tealium.crashreporter.CrashReporter.Tracker
+import com.tealium.internal.data.Dispatch
+import com.tealium.internal.listeners.ActivityResumeListener
+import com.tealium.internal.listeners.DisableListener
+import com.tealium.internal.listeners.PopulateDispatchListener
+import com.tealium.library.DataSources
+import com.tealium.library.Tealium
+
+/**
+ * Crash Reporter module for tracking crash data. When used, this module populates
+ * <i>crash_*</i> attributes defined in {@link com.tealium.library.DataSources.Key}.
+ */
+class CrashReporter (config: Tealium.Config, tracker: Tracker, truncateStackTrace: Boolean) : Thread.UncaughtExceptionHandler {
+
+    const val CRASH_COUNT =  "crash_count"
+
+    val mOriginalExceptionHandler: Thread.UncaughtExceptionHandler
```

Comment:
all of these don't need to be public as far as I can see, so would be best to have them `private`